src/styles/themes.less

```
@@ -80,3 +162,15 @@
     filter: invert(1) brightness(90%);
   }
 }
+
+.united {
+    img[src="assets/images/dungeons/boss.svg"] {
+        filter: invert(1) brightness(90%);
+    }
+}
+
+.yeti {
+    img[src="assets/images/dungeons/boss.svg"] {
+        filter: invert(1) brightness(90%);
+    }
+}
```

All of these can be combined into one like this:
```suggestion
.spacelab,
.united,
.yeti {
    img[src="assets/images/dungeons/boss.svg"] {
        filter: invert(1) brightness(90%);
    }
}
```